src/lib/tls/tls_policy.cpp

```diff
@@ -161,31 +161,53 @@ Group_Params Policy::default_dh_group() const {
 }
 
 std::vector<Group_Params> Policy::key_exchange_groups() const {
-   // Default list is ordered by performance
    return {
+      // clang-format off
+#if defined(BOTAN_HAS_X25519) && defined(BOTAN_HAS_KYBER_ROUND3) && defined(BOTAN_HAS_TLS_13_PQC)
+      Group_Params::HYBRID_X25519_KYBER_768_R3_OQS,
+#endif
```

Comment:
Probably, now this should offer ML-KEM instead. I'm expecting the community to move away from the pre-standard algorithm fairly quickly.

Reply:
Concur, especially since this change isn't going to be going out until February